config/prod/iatlas-db.yaml

```diff
@@ -0,0 +1,18 @@
+template_path: aurora-postgresql-db.yaml
+stack_name: iatlasdb-prod
+dependencies:
+  - common/iatlasvpc.yaml
+  - prod/iatlas-kms-prod.yaml
+stack_tags:
+  Department: "CompOnc"
+  Project: "iAtlas"
+  OwnerEmail: "andrew.lamb@sagebase.org"
+parameters:
+  VpcName: iatlasvpc
```

Comment:
use sceptre !Stack_output reference here?

Reply:
It's possible... though when I tried, again, the VpcName is used to reference multiple stack outputs, so its shorter this way.

Reply:
ok